main.js

```diff
@@ -35,7 +35,7 @@ const isBrowser = (typeof self !== 'undefined');
  */
 export async function signCapabilityInvocation({
   url,
-  method,
+  method = '',
```

Comment:
Can we leave this alone? Even if it isn't a string, we're going to have trouble in the error message generation in `catch` if someone passes in something that isn't one. We should change that message generation code, I'll suggest something else.

```suggestion
  method,
```

Reply:
Sounds good, yeah.